homeassistant/components/camera/__init__.py

```diff
@@ -777,6 +784,47 @@ async def handle(self, request: web.Request, camera: Camera) -> web.StreamRespon
             raise web.HTTPBadRequest() from err
 
 
+class Annotation:
+    """Text to be added to snapshot image."""
+
+    annotation_padding: int = 10
+
+    def __init__(self, service_call: ServiceCall, hass: HomeAssistant) -> None:
+        """Initialize annotation with ServiceCall and Hass."""
+        if "annotation" in service_call.data:
+            self.text = service_call.data["annotation"]
+            self.hass = hass
+            self.is_defined = True
+        else:
+            self.is_defined = False
+
+    def annotate(self, image_stream:bytes) -> bytes:
+        """Write annotation to the file provided as byte stream direct from camera feed."""
+        self.text.hass = self.hass
+        annotation_text = self.text.async_render()
+        _LOGGER.debug(
+            "Annotating snapsot with template text: %s",
```

Comment:
Typo here and probably should remove the word 'template' as this message will print the rendered template, not the template itself.

Reply:
done